src/storage/gpu_swap_history.cc

```diff
@@ -145,12 +144,55 @@ handle_id_t MemHistory::NaiveHistoryBased(
 
 }
 
-handle_id_t MemHistory::DecideVictim(std::unordered_set<handle_id_t> handles, int device) {
+handle_id_t MemHistory::SizeHistory(
+    std::unordered_set<handle_id_t> handles, int device, void* arg) {
+  auto divided_handles  = ((SwapParams*)arg)->divided_handles;
+  auto candidates = divided_handles->lower_bound(((SwapParams*)arg)->required_memory);
+  auto original_candidates = candidates;
+  bool reverse_flag = false;
+  size_t no_swap_step = 80;
```

Comment:
Add "FIXME: Empirical result may need a better way to know how to choose this." before this line.